.mdlrc

```diff
@@ -0,0 +1 @@
+rules "~MD005", "~MD033", "~MD046"
```

Comment:
33 will complain about embedded HTML and didn't like `<sup>`, so that had to go.
The other two were too picky and hard to satisfy.